testing/pypy3/APKBUILD

```diff
@@ -0,0 +1,48 @@
+# Contributor: Mihnea Saracin <mihnea.saracin@rinftech.com>
+# Maintainer: Mihnea Saracin <mihnea.saracin@rinftech.com>
+pkgname=pypy3
+pkgver=5.7.1
+pkgrel=0
+pkgdesc="A fast, compliant alternative implementation of the Python 3 language"
+url="https://github.com/HorridBro/AlpinePyPy3"
+arch="all"
+license="MIT"
+depends=""
+makedepends="python linux-headers libffi-dev pkgconf zlib-dev bzip2-dev sqlite-dev ncurses-dev readline-dev libressl-dev expat-dev gdbm-dev paxmark tk tk-dev py3-cffi py2-cffi xz xz-dev"
+subpackages="$pkgname-dev"
+source="https://bitbucket.org/pypy/pypy/downloads/$pkgname-v$pkgver-src.tar.bz2
+        alpine.patch libpthread_version.patch const_stdio.patch"
+builddir="$srcdir/$pkgname-v$pkgver-src"
+
+
+
+build() {
+        cd "$builddir/pypy/goal"
+        python2 ../../rpython/bin/rpython --opt=jit --nopax targetpypystandalone.py 
+}
+
+
+
+package() {
+        cd "$builddir/pypy/tool/release"
+        ./package.py --archive-name $pkgname-$pkgver --builddir "$pkgdir" 
+}
+
+check(){
+	cd "$builddir/lib-python/3/test/"
+	# XXX: Some tests fail because they are implemented using CPython C functions
+	$builddir/pypy/goal/pypy3-c regrtest.py -q || true
+}
```

Comment:
`check` runs between `build` and `package`, so please put it between them, to make it more clear.